Formula/ripgrep.rb

```diff
@@ -15,11 +15,13 @@ class Ripgrep < Formula
   depends_on "asciidoc" => :build
   depends_on "docbook-xsl" => :build
   depends_on "rust" => :build
+  depends_on "pkg-config" => :build
```

Comment:
Sorry to be picky but can you please place these dependencies in alphabetical order ? Thanks!

Reply:
Done

Reply:
I just looked into why brew audit didn't catch this sooner. I ran just `brew audit --strict ripgrep` but I didn't know to run `brew audit --new-formula --strict ripgrep` as that would've caught the problem sooner.

Reply:
Aye. It's another of those balancing acts between wanting things done a certain way but not wanting to break a significant chunk of existing formulae passing through CI for other reasons.